modules/multiplayer/scene_cache_interface.cpp

```diff
@@ -204,6 +207,28 @@ Error SceneCacheInterface::_send_confirm_path(Node *p_node, NodeCache &p_cache,
 	return err;
 }
 
+ObjectID SceneCacheInterface::resolve_object_id(int p_id) const {
+	if (assigned_ids.has(p_id)) {
+		return assigned_ids.get(p_id);
+	}
+
+	return ObjectID{ (uint64_t)0 };
+}
+
+int SceneCacheInterface::get_network_id(const Object *p_object, int p_peer_id) const {
+	ERR_FAIL_NULL_V(p_object, 0);
+
+	ObjectID oid = p_object->get_instance_id();
+	const NodeCache *nc = nodes_cache.getptr(oid);
+
+	// if (!nc->confirmed_peers.has(p_peer_id) || !nc->confirmed_peers.get(p_peer_id) || !nc->recv_ids.has(p_peer_id)) {
```

Comment:
Does it make sense to check here if `p_peer_id` has confirmed the remote ID?

Reply:
If the peer has not confirmed the id? Is there anything the user can do? Or can the RPC be queued until the id has been confirmed?

Reply:
Reworked this a bit. Now getting the ID will always return the locally assigned ID, and an extra method was added to check if that ID was confirmed by a given peer.